source/MaterialXTest/Types.cpp

```diff
@@ -31,7 +32,14 @@ TEST_CASE("Vectors", "[types]")
     REQUIRE((v2 /= v1) == mx::Vector3(2, 4, 6));
     REQUIRE(v1 * 2 == v2);
     REQUIRE(v2 / 2 == v1);
+    
+    // Geometric operators
+    mx::Vector3 v3(1.0f, 2.0f, 2.0f);
```

Comment:
For unit tests where we're restricting ourselves to integers that are exactly representable as single-precision floats, I would recommend omitting the ".0f" extension, to highlight the integer nature of these values.  Naturally, this won't be possible for tests of rotation matrices, but then we can use your new "equivalent" method to highlight the fact that we're comparing arbitrary floating-point values.